src/targets/javascript/fetch/fixtures/multipart-file.js

```diff
@@ -6,6 +6,6 @@ const options = {method: 'POST'};
 options.body = form;
 
 fetch('https://httpbin.org/anything', options)
```

Comment:
These snippets, and the JS `fetch` target client itself, still diverge a bit from the newly revised Node `fetch` one because the Node one needs to actually be able to support file uploads, which short of importing `fs` into these JS snippets can't be done.

Makes me wonder how much value we really get from these plain JS snippets now that Node has converged with JS entirely on the `fetch` front. Axios Node and JS work the same, do we really need two entirely separate client generators, with identical clients, for the same language? Should we drop JS as an available target in favor of Node?